### PR TITLE
feat: Add count rule for api invalid uris

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Use waffles to find the uris:
```
_status
api-key
complaint
email-branding
events
inbound-number
invite
letter-branding
letters
notifications
organisation
organisations
platform-stats
provider-details
service
static
user
v2
```

```
python waffles.py list  --app-loc ~/Notify/notification-api --app-lib ~/.virtualenvs/notification-api/lib/python3.9/site-packages --flask-mod application --flask-prop application \
| grep ",OPTIONS"  \
|  sed 's/^[^\/]*\///' \
| sed 's/\/.*//' \
| sort | uniq
```

To test by monitoring in staging

### CODE REVIEW

File: aws/eks/waf.tf [hunk 1]
```diff
@@ -290,6 +290,105 @@ resource "aws_wafv2_web_acl" "notification-canada-ca" {
     }
   }
 
+  rule {
```
Comment (sastels) [inline aws/eks/waf.tf:293]: tbh, maybe we just put this in the api lambda's WAF? that's where users (ie ontside k8s) will go anyway.

Then also don't need to check the hostname.

**Review by sastels — APPROVED** [review]
Let's see what this does in staging!

**Comment by jimleroyer** [comment]
Good stuff!

**Comment by jzbahrai** [comment]
Screen Shot 2022-08-30 at 4 59 58 PM

**Pull request merged**